hack/kind-up.sh

```diff
@@ -220,6 +220,28 @@ setup_kind_with_lpp_resize_support() {
   kubectl delete --ignore-not-found=true storageclass local-path
 }
 
+check_shell_dependencies() {
+  if ! sed --version  >/dev/null 2>&1; then
+    echo "WARNING: Your sed version does not support the --version flag. Please ensure you have a compatible version of GNU sed installed."
```

Comment:
I think we could go one step further and exit with an error. This warning might be easy to overlook in the flood of logs. 😁